### ISSUE TITLE
Require ([..]/autoload.php) failed

### ISSUE THREAD

**Issue by BenjaminBeck** [AOEpeople/TYPO3_Restler#9]
After installing in a pure composer installation with 7.6.6 i got the error (when opening the /api_explorer):

``Warning: require(/var/www/domains/[...]/web/typo3conf/ext/restler/Classes/System/TYPO3/../../../../../../../typo3_src/vendor/autoload.php): failed to open stream: No such file or directory in /var/www/domains/[...]/web/typo3conf/ext/restler/Classes/System/TYPO3/Loader.php on line 140``

After i did the following change in ``/web/typo3conf/ext/restler/Classes/System/TYPO3/Loader.php``:
``$classLoader = require __DIR__ . '/../../../../../../../typo3_src/vendor/autoload.php';`` (``L:140``)
to
``$classLoader = require __DIR__ . '/../../../../../../../vendor/typo3/cms/vendor/autoload.php';``
it worked.

I have this in my composer.json:
```json
"extra": {
    "typo3/cms": {
      "cms-package-dir": "{$vendor-dir}/typo3/cms",
      "web-dir": "web"
    }
  }
```

**Comment by vdespa** [comment]
@BenjaminBeck Thanks for your bug report.

I think is was already reported (see #6) and fixed in 1.2.21.

Which version are you using and how are you loading the restler extension (TER, composer)?

**Comment by BenjaminBeck** [comment]
@vdespa You are right - i found a problem on my side:
My version was pretty old: 1.2.13.
Sorry for not checking the old issues..

I put ``"aoe/restler": "1.*"`` into my root composer.json and expected it to pull the latest version (global minimum-stability is dev).
Maybe there is a old version somewhere in the typo3 composer repo? (http://composer.typo3.org/)

When setting ``"aoe/restler": "dev-master@dev"`` and adding the git repo directly it pulls the latest version.

Thanks for your help
